controller/space_iterations_test.go

```diff
@@ -90,8 +91,11 @@ func (rest *TestSpaceIterationREST) TestSuccessCreateIteration() {
 	ci := createSpaceIteration("Sprint #21", nil)
 	err := application.Transactional(rest.db, func(app application.Application) error {
 		repo := app.Spaces()
+		spaceTemplate, err := spacetemplate.CreateEmptySpaceTemplateWithRepo(context.Background(), app.SpaceTemplates())
```

Comment:
Because it is used in many packages (`spacetemplate`, `area`, `controller`, `workitem`, `migration`, `space`) and I felt it is best put in the `spacetemplate` package.